Makefile

```diff
@@ -0,0 +1,18 @@
+ifeq ($(shell which miniwdl),)
+$(error Please install requirements using "pip install -r requirements.txt")
+endif
+
+ifeq ($(shell which shellcheck),)
```

Comment:
Does miniwdl require shellcheck now?  I'm not sure where this is being used.

Reply:
It's in the [docs](https://github.com/chanzuckerberg/miniwdl/blob/5b66f2f0b5101a70bae9df51bc7772a93189ba51/docs/check.rst)

Reply:
Might be good to include in the docs for this project.  Or at least a comment.  There's no indication in this codebase that it's used for miniWDL.

Reply:
added comment